### PR TITLE
Clean up README for 2.1 release branch.

### PR DESCRIPTION
**Pull request by will-cromar**
- Remove CI status badge
- Refer to main README for list of all build versions
- Change `libtpu_releases.html` to `index.html` (also update corresponding setup.py comment)
- Fix some broken relative links

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -356,7 +356,7 @@ def run(self):
     },
     extras_require={
         # On Cloud TPU VM install with:
-        # pip install torch_xla[tpu] -f https://storage.googleapis.com/libtpu-releases/libtpu_releases.html
+        # pip install torch_xla[tpu] -f https://storage.googleapis.com/libtpu-releases/index.html
```
Comment (JackCaoG) [inline setup.py:359]: why did this changed?

Comment (will-cromar) [reply setup.py:359]: This is a minor implementation detail. `libtpu_releases.html` and `index.html` are almost identical, except `index.html` is not populated with libtpu versions on the days when the build is failing. `libtpu_releases.html` has a line for every day that points to the last passing build, which may be days old if libtpu is broken.

Comment (JackCaoG) [reply setup.py:359]: should we update this to the master as well?

**Review by JackCaoG — APPROVED** [review]

File: setup.py (hunk 1 above)
Comment (will-cromar) [reply setup.py:359]: Yeah, I'll backport the setup.py changes to master

**Pull request merged**